src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -25,6 +25,7 @@
   @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<Application> application;
```

Comment:
Wondering if we want to include `review` field?
It is available in the openAPI, and in the pay-server code
https://git.corp.stripe.com/stripe-internal/pay-server/blob/master/lib/payment_flows/api/payment_intent/resource.rb#L85-L86